tests/std/tests/P0896R4_views_iota_death/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\strict_concepts_20_matrix.lst

```

Comment:
Do we really need the `strict` matrix? The `iota_view` test notably doesn't, and the pertinent iterators don't have `iter_move` or `iter_swap` friends.